src/app.tsx

```diff
@@ -64,6 +64,19 @@ let prereqEditing = (window as Window)
     ? 'Page prerequisite editing enabled' : 'Page prerequisite editing disabled';
 };
 
+let foreignLanguageAccessibility = (window as Window)
+  .localStorage.getItem('foreign-language-accessibility') === 'true';
+(window as any).enableForeignLanguageAccessibility = (show: boolean) => {
+  foreignLanguageAccessibility = !!show;
+  (window as Window).localStorage.setItem(
+    'foreign-language-accessibility',
+    foreignLanguageAccessibility ? 'true' : 'false');
+
+  return foreignLanguageAccessibility
+    ? 'Foreign language accessibility support enabled'
+    : 'Foreign language accessibility support disabled';
+};
+
```

Comment:
Add foreign language support to console with
`enableForeignLanguageAccessibility(enable: boolean)`